client/my-sites/checkout/checkout/source-payment-box.jsx

```diff
@@ -0,0 +1,189 @@
+				if ( error.message ) {
+					errorMessage = error.message;
+				} else {
+					errorMessage = translate( 'Please specify a country and postal code.' );
+				}
+
+				this.setSubmitState( {
+					error: errorMessage,
+					disabled: false
+				} );
+			}
+
+			if ( sourcePaymentUrl ) {
+				this.setSubmitState( {
+					info: translate( 'Redirecting you to our payment provider' ),
+					disabled: true
+				} );
+				analytics.ga.recordEvent( 'Upgrades', 'Clicked Checkout With Source Payment Button' );
+				analytics.tracks.recordEvent( 'calypso_checkout_with_source_' + this.props.paymentType );
+				location.href = sourcePaymentUrl;
+			}
+		}.bind( this ) );
+	}
+
+	renderButtonText() {
+		if ( cartValues.cartItems.hasRenewalItem( this.props.cart ) ) {
+			return translate( 'Purchase %(price)s subscription with %(paymentProvider)s', {
+				args: { price: this.props.cart.total_cost_display, paymentProvider: this.getPaymentProviderName() },
+				context: 'Pay button on /checkout'
+			} );
+		}
+
+		return translate( 'Pay %(price)s with %(paymentProvider)s', {
+			args: { price: this.props.cart.total_cost_display, paymentProvider: this.getPaymentProviderName() },
+			context: 'Pay button on /checkout'
+		} );
+	}
+
+	content() {
+		const hasBusinessPlanInCart = some( this.props.cart.products, { product_slug: PLAN_BUSINESS } );
+		const showPaymentChatButton =
+			config.isEnabled( 'upgrades/presale-chat' ) &&
+			abtest( 'presaleChatButton' ) === 'showChatButton' &&
+			hasBusinessPlanInCart;
+
+		return (
+			<form onSubmit={ this.redirectToPayment }>
+
+				<div className="payment-box-section">
+					<Input
+						additionalClasses="checkout-field"
+						name="name"
+						label={ translate( 'Name', { textOnly: true } ) }
+						eventFormName="Checkout Form" />
+				</div>
+
+				<TermsOfService
+					hasRenewableSubscription={ cartValues.cartItems.hasRenewableSubscription( this.props.cart ) } />
+
+				<div className="payment-box-actions">
+					<div className="pay-button">
+						<button type="submit" className="button is-primary button-pay" >
+							{ this.renderButtonText() }
+						</button>
+						<SubscriptionText cart={ this.props.cart } />
+					</div>
+
+					<AlternativePaymentMethods
+						cart={ this.props.cart }
+						paymentMethods={ this.props.paymentMethods }
+						selectedPaymentMethod={ this.props.paymentType }
+						onSelectPaymentMethod={ this.props.onSelectPaymentMethod }
+						/>
+
+					{
+						showPaymentChatButton &&
+						<PaymentChatButton
+							paymentType={ this.props.paymentType }
+							cart={ this.props.cart } />
+					}
+				</div>
+
+				<CartCoupon cart={ this.props.cart } />
+
+				<CartToggle />
+			</form>
+		);
+	}
+
+	getPaymentProviderName() {
+		switch ( this.props.paymentType ) {
+			case 'ideal':
+				return 'iDEAL';
+		}
+
+		return this.props.paymentType;
+	}
+
+	getTitle() {
+		switch ( this.props.paymentType ) {
+			case 'ideal':
+				return translate( 'Secure Payment with iDEAL' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'Secure Payment with PayPal' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/payment-ideal)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).